### PR TITLE
Adds ref to the wiki for Swift/AWS config

### PR DESCRIPTION
**Pull request by lamagnifica**
@nedbat please take a quick sanity check of the linked wiki page

@feanil FYI

@pdesjardins or @srpearce sanity check please

### CODE REVIEW

**Comment by lamagnifica** [comment]
test pass loally

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 1]
```diff
@@ -121,9 +121,9 @@ retrieval, the Open edX platform now uses the Django file storage API. In
 addition to S3, you can now configure an OpenStack Swift Object Store as the
 cloud storage provider for certain system features.
 
-Documentation updates for this feature are currently in progress.
+For more information, see the `Configuring Data Storage`_ wiki page..
```
Comment (pdesjardins) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: Is that a double period?

**Comment by pdesjardins** [comment]
Minor comment, 👍

File: en_us/open_edx_release_notes/source/eucalyptus.rst (hunk 1 above)
Comment (lamagnifica) [reply en_us/open_edx_release_notes/source/eucalyptus.rst]: oops

**Comment by nedbat** [comment]
:+1:

**Pull request merged**